code/game/objects/effects/decals/crayon.dm

```diff
@@ -5,10 +5,12 @@
 	layer = 2.1
 	anchored = 1
 
-/obj/effect/decal/cleanable/crayon/New(location,main = "#FFFFFF",shade = "#000000",var/type = "rune")
+/obj/effect/decal/cleanable/crayon/New(location,main,shade,var/type="rune")
 	..()
 	loc = location
 
+	if(!main || !shade || !type) return

```

Comment:
oh shit I accidentally did that while working on custom text and forgot to remove it later